### PR TITLE
HOMS-239 Fix BP name and add BP source code

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

**Comment by mxsmirnoff** [comment]
Updated

File: demo_processes/pizza-order-camunda/src/main/resources/scripts/init.groovy [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//Load app parameters from ENV variables
+def ENV = System.getenv()
+
+// camunda-ext
+execution.setVariable("homsUrl",     "http://homs:3000")
```
Comment (sugarfree1) [inline demo_processes/pizza-order-camunda/src/main/resources/scripts/init.groovy]: Use ```HOMS_HOST``` and ```HOMS_PORT```.

**Comment by mxsmirnoff** [comment]
Updated

File: demo_processes/pizza-order-camunda/src/main/resources/scripts/init.groovy [hunk 2]
```diff
@@ -0,0 +1,8 @@
+//Load app parameters from ENV variables
+def ENV = System.getenv()
+
+// camunda-ext
+execution.setVariable("homsUrl",     "http://${ENV['HOMS_HOST']}:${ENV['HOMS_PORT']}")
```
Comment (sugarfree1) [inline demo_processes/pizza-order-camunda/src/main/resources/scripts/init.groovy]: Add one space before ```"```

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**